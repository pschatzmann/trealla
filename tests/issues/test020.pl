:-initialization(main).

main :-
	_X = 3s,
	true.
