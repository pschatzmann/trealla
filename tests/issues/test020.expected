Error: syntax error, near 's', operator expected '	_X = 3s,
'
